.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Pipeline
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  Metamodel-Modeling-Foundations:
+    uses:  MDSD-Tools/Palladio-Build-ActionsPipeline/.github/workflows/build.yml@v3.2
+    with:
+      use-display-output: true
+      no-caching: true
+      java-version: 17
+      runner-label: ubuntu-latest
+      deploy-updatesite: 'releng/tools.mdsd.probdist.updatesite/target/repository'
```

Comment:
This is the wrong path. If you have a look in the old jenkinsfile you can see it should be:
'releng/tools.mdsd.modelingfoundations.updatesite/target/repository'

Reply:
done